zbus-lockstep/src/macros.rs

```diff
@@ -26,15 +26,16 @@ use crate::Result;
 /// std::env::set_var("LOCKSTEP_XML_PATH", "../xml");
 ///
 /// let xml_path = resolve_xml_path(None).unwrap();
-/// assert_eq!(xml_path, PathBuf::from("../xml").canonicalize().unwrap());       
+/// assert_eq!(xml_path, PathBuf::from("../xml").canonicalize().unwrap());
 /// # }
 /// ```
 /// # Panics
 ///
 /// Panics if no XML path is provided and the default XML path is not found.
 pub fn resolve_xml_path(xml: Option<&str>) -> Result<PathBuf> {
     let mut xml = xml;
-    let current_dir: PathBuf = std::env::current_dir()?;
+    let current_dir: PathBuf =
+        PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR is not set"));
```

Comment:
Thanks for the PR! Much appreciated!

Could you adjust the `expect` message to say why we expect it to succeed? See also [the recommended message style on `Option`](https://doc.rust-lang.org/stable/std/option/enum.Option.html#recommended-message-style).

Reply:
Done!